recipes/fwdpy/meta.yaml

```diff
@@ -0,0 +1,69 @@
+package:
+  name: fwdpy
+  version: "0.0.4pre1"
+
+source:
+  fn: fwdpy-0.0.4pre1.tar.gz
+  url: https://pypi.python.org/packages/ac/8a/01a4a85d20a924abfc981a57aba9b364d6f7e00a0830c5e654242afc9c39/fwdpy-0.0.4pre1.tar.gz
+  md5: f3377dfd2ec9e84beba2d0edcedb8772
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # noarch_python: True
+  # preserve_egg_dir: True
+  #entry_points:
+  #  - mspms=msprime.cli:mspms_main
+  #  - msp=msprime.cli:msp_main
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  number: 0
+  string: "py{{CONDA_PY}}_gsl{{CONDA_GSL}}_{{PKG_BUILDNUM}}"
+
+requirements:
+  build:
+    - gcc
+    - python
+    - libsequence
+    - pandas
+    - numpy
+    - fwdpp >=0.5.4
+    - gsl {{CONDA_GSL}}*
+
+  run:
+    - python
+    - numpy
+    - numpy
```

Comment:
two times numpy, you probably also want to build it against multiple numpy versions?